### ISSUE TITLE
Mark this repo as deprecated

### ISSUE THREAD

**Issue by nlharris** [monarch-initiative/monarch-legacy#1650]
Should we add something to the README for this repo (and possibly in other places as well, like can we add something that shows up at the top of the issue tracker?) that notes that this repo is obsolete (it's for the old, soon-to-be-retired website) and has been superseded by https://github.com/monarch-initiative/monarch-ui?

**Comment by nicolevasilevsky** [comment]
yes!

**Comment by monicacecilia** [comment]
Yes, this should be done eventually. Thanks for issuing this note. 

Since the current website is still the source of truth for many resources, and this repository is still actively being used for deployment of the current website, I am more inclined to add the 'deprecated' note as we near the release of the new UI, and of this issue, Strategy for shipping new monarch-ui https://github.com/monarch-initiative/monarch-ui/issues/91.

**Comment by monicacecilia** [comment]
I added a note on the UI Project Board, [here](https://github.com/orgs/monarch-initiative/projects/8#card-23716100).
